projects/cloud/app/graphql/mutations/change_user_role.rb

```diff
@@ -0,0 +1,17 @@
+module Mutations
+  class ChangeUserRole < ::Mutations::BaseMutation
+    argument :user_id, ID, required: true
+    argument :organization_id, String, required: true
+    argument :role, Types::RoleType, required: true
+
+    type Types::UserType
+
+    def resolve(attributes)
+      begin
+        ChangeUserRoleService.call(**attributes, acting_user: context[:current_user])
+      rescue ChangeUserRoleService::Error::Unauthorized
+        raise GraphQL::ExecutionError, "You do not have a permission to change a role for this user."
```

Comment:
I'm afraid this will result in a non-standard error format that might change if `ruby-graphq` changes the way errors are managed. I'd instead introduce a new type that models errors, and return that instead. I'd leave `GraphQL::ExecutionError` only for uncaught errors.